hknweb/events/models.py

```diff
@@ -1,66 +1,53 @@
 from django.db import models
+from hknweb.models import Profile
 
 class Event(models.Model):
-    id          = models.IntegerField(primary_key=True)
     name        = models.CharField(max_length=255, null=False)
     slug        = models.CharField(max_length=255)
     location    = models.CharField(max_length=255)
     description = models.TextField()
     start_time  = models.DateTimeField(null=False)
     end_time    = models.DateTimeField(null=False)
-    event_type_id = models.IntegerField()
+    rsvp_limit  = models.PositiveIntegerField(null=True, default=999999) # I'm being rather pessimistic in assuming no HKN event will hit 1 million participants!
+    rsvps       = models.IntegerField(default=0)
+    # event_type_id = models.IntegerField()
     # event_type_id  = models.ForeignKey()
```

Comment:
This will tie in with a larger design, but I think we also need an EventType model, with every Event containing a ForeignKey pointing to an EventType. This way candidates will know which event is a Big Fun event, vs. a Prodev or Serv or Fun event. (The idea is that each of these event types is an EventType object stored in the database.)